view/simple/src/main/kotlin/io/holunda/polyflow/view/simple/service/Converters.kt

```diff
@@ -66,6 +64,20 @@ fun DataEntryCreatedEvent.toDataEntry() = DataEntry(
   protocol = listOf<ProtocolEntry>().addModification(this.createModification, this.state)
 )
 
+/**
+ * Event to entry for an update, if an optional entry exists.
+ */
+fun DataEntryAnonymizedEvent.toDataEntry(oldEntry: DataEntry) =
+  oldEntry.copy(
+    authorizedUsers = AuthorizationChange.applyUserAuthorization(oldEntry.authorizedUsers,
+      oldEntry.authorizedUsers.map { AuthorizationChange.removeUser(it) }),
+    protocol = oldEntry.protocol.map {
+      it.copy(
+        username =
+        if (it.username != null && !this.excludedUsernames.contains(it.username))
+          this.anonymizedUsername else it.username) }
+      .addModification(this.anonymizeModification, oldEntry.state))
```

Comment:
Same as in JPA and Mongo, please first add the modification and then anonymize

Reply:
same